src/sage/categories/pushout.py

```diff
@@ -3545,11 +3550,9 @@ def merge(self, other):
         # integers to encode degrees of extensions.
         from sage.rings.integer import Integer
         kwds_self = dict(self.kwds.items())
-        if 'impl' in kwds_self:
-            del kwds_self['impl']
+        if 'implementation' in kwds_self:
+            del kwds_self['implementation']
```

Comment:
L3553,3554 could also be deleted, I think.

Reply:
Removed.